src/lib/components/Tree.vue

```diff
@@ -98,7 +100,7 @@ export default {
     const { searchTree } = useSearch()
     const reactiveNodes = ref(props.nodes)
     const reactiveExpandRowByDefault = ref(props.expandRowByDefault)
-
+    const isIndeterminate = reactive({ status: false })
```

Comment:
If it will always only hold status, it may be better to keep a Boolean value directly.

```js
const isIndeterminate = ref( false )
```